code/game/objects/items/stacks/medical.dm

```diff
@@ -90,29 +94,38 @@
 
 	if(!affecting) //Missing limb?
 		to_chat(user, "<span class='warning'>[C] doesn't have \a [parse_zone(zone_selected)]!</span>")
+		C.balloon_alert(user, "[C] has no [parse_zone(zone_selected)]!")
 		return
 
-	if(ishuman(C)) //apparently only humans bleed? funky.
-		var/mob/living/carbon/human/H = C
-		if(stop_bleeding)
-			if(!H.bleed_rate)
-				to_chat(user, "<span class='warning'>[H] isn't bleeding!</span>")
-				return
-			if(H.bleedsuppress) //so you can't stack bleed suppression
-				to_chat(user, "<span class='warning'>[H]'s bleeding is already bandaged!</span>")
-				return
-			H.suppress_bloodloss(stop_bleeding)
+	var/valid = FALSE
+	var/message = null
+
+	if(stop_bleeding)
```

Comment:
In-game you can bandage when there's no bleeding and even when it's already bandaged. There might be an issue here but idk how to check

Reply:
This is due to the fact that gauzes claim to heal brute damage so that they can be used to heal simpleanimals. I have fixed this in the most recent commit.